packages/types/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "../../tsconfig.node.json",
   "include": ["src", "test"],
   "compilerOptions": {
+    "rootDir": "./src",
```

Comment:
ahh, everytime I’ve had to add this, something else was actually off. what’s wrong if we remove that?

Reply:
really? I thought this puts all the types in the correct folders?

Reply:
removed it lets see